src/app/info/supabaseCall.ts

```diff
@@ -0,0 +1,48 @@
+import supabase from "@/supabaseClient";
+import { PostgrestError } from "@supabase/supabase-js";
+import { WikiRowQueryType } from "@/app/info/AddWikiItemButton";
+
+export async function reorderSupabaseCall(
+    key1: string,
+    key2: string
+): Promise<PostgrestError | null> {
+    const { error } = await supabase.rpc("swap_two_wiki_rows", {
+        key1: key1,
+        key2: key2,
+    });
+
+    return error;
+}
```

Comment:
Really nice refactoring here! Can we change the names a little bit? I think calling the file `supabaseHelpers.ts` describes the contents of the file a little better. And for the function names, something like `updateSupabaseCall` suggests to me that there is a call to supabase somewhere and we are updating that call before we send it off, not that we're making an update call to supabase. I think something like `callUpdateSupabase` or even `updateWikiDatabase` makes the intention a bit clearer. Can we do something like that for all the function names?